node/sc/bridge_manager.go

```diff
@@ -46,6 +46,11 @@ const (
 	ERC721
 )
 
+const (
+	voteTypeValueTransfer = 0
+	voteTypeConfiguration = 1
```

Comment:
Why thises variables are not declared in `multi_bridge_test.go`, but `bridge_manager.go`?

Reply:
If we add api for some bridgeOperator contract methods later, the constants are necessary.

Reply:
Okay. Thank you.